build.gradle

```diff
@@ -116,6 +116,7 @@ tasks.register('afterEclipseBuildshipImport', Exec) {
 
 googleJavaFormat {
 	toolVersion = '1.7'
+	exclude 'com.ibm.wala.cast.java.test.data/**/*.java'
```

Comment:
We exclude the test data for cast since there are assertions in our tests based on source positions.  Eventually if we like we can reformat this code and also clean up those assertions, though it doesn't seem that important to me.

Reply:
Sounds reasonable. Worth explaining in a comment or as part of a commit message?

Reply:
Yup, good call.  Added a comment.